app/README.md

```diff
@@ -1,12 +1,5 @@
-# Wegue App Dir
-This directory will contain your Wegue Application.
-By using the command `npm run init:app` this directory is populated with
-the contents from the `app-starter/` dir.
+# Wegue App Starter Dir
```

Comment:
I think this is somehow unrelated and even wrong since you are overiting the README in the `app/` folder with the content of `app-starter`, right?

Reply:
Right. I reverted it with 8811c22a12549cda8e88b8ab16e65547c60e0ae0